models/DataObject/ClassDefinition.php

```diff
@@ -1242,10 +1242,13 @@ public function getGroup()
 
     /**
      * @param string $group
+     *
+     * @return $this
      */
     public function setGroup($group)
     {
         $this->group = $group;
+        return $this;
```

Comment:
Missing blank line before `return`:

```suggestion

        return $this;
```